tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,613 @@
+        spec:
+            containers:
+            - name: "elasticsearch"
+            env:
+            - name: "ZONE"
+                valueFrom:
+                fieldRef:
+                    fieldPath: "metadata.annotations['topology.kubernetes.io/zone']"
+            topologySpreadConstraints:
+            - maxSkew: 1
+            topologyKey: "topology.kubernetes.io/zone"
+            whenUnsatisfiable: "DoNotSchedule"
+            labelSelector:
+                matchLabels:
+                "elasticsearch.k8s.elastic.co/cluster-name": "multi-az"
+                "elasticsearch.k8s.elastic.co/statefulset-name": "multi-az-es-default"
+    YAML
+    }
+    ```
+
+4. Then we can deploy a Kibana instance, which will be automatically configured to connect to the Elasticsearch cluster. We use the same `topologySpreadConstraints` as for Elasticsearch to spread the pods across the different AZs. We also use the `eck.k8s.elastic.co/cluster-name` annotation to tell the operator to connect this Kibana instance to the Elasticsearch cluster named "multi-az" created before.
+    ```
+    resource "kubectl_manifest" "kibana" {
+    yaml_body = <<YAML
+    apiVersion: "kibana.k8s.elastic.co/v1"
+    kind: "Kibana"
+    metadata:
+    name: "kibana"
+    namespace: "${kubernetes_namespace.elasticsearch.metadata[0].name}"
+    spec:
+    version: "8.10.2"
+    count: 3
+    elasticsearchRef:
+        name: "multi-az"
+    http:
+        tls:
+        selfSignedCertificate:
+            disabled: true
+    YAML
+    }
+    ```
+
+5. To expose Kibana to the outside world, we can create an Ingress resource that matches the host `kibana.scw.your-domain.tld` and redirect traffic to the Kibana service. Once applied it will allow you to access Kibana on https://kibana.scw.your-domain.tld using your browser.
+    ```
+    resource "kubernetes_ingress_v1" "kibana" {
+    metadata {
+        name      = "kibana"
+        namespace = kubernetes_namespace.elasticsearch.metadata[0].name
+    }
+
+    spec {
+        ingress_class_name = "nginx"
+        rule {
+        host = "kibana.scw.your-domain.tld"
+        http {
+            path {
+            backend {
+                service {
+                name = "kibana-kb-http"
+                port {
+                    number = 5601
+                }
+                }
+            }
+            }
+        }
+        }
+    }
+    }
+    ```
+
+6. Finally, create a DNS CNAME record for `kibana.scw.your-domain.tld`, pointing to the A record of your ingress-nginx controller that deployed in the second section. Since this A record ,`ingress.scw.your-domain.tld`, is configured with health checks, it will automatically resolve to the IPs of the available load balancers, so in case of the loss of one AZ, the DNS record will automatically resolve to the IP of the healthy load balancer and the traffic will be dispatched to the healthy nodes of the other AZs, maintaining the availability of Kibana.
```

Comment:
```suggestion
6. Finally, create a DNS CNAME record for `kibana.scw.your-domain.tld`, pointing to the A record of your ingress-nginx controller deployed in the second section. Since this A record ,`ingress.scw.your-domain.tld`, is configured with health checks, it will automatically resolve to the IPs of the available load balancers, so in case of the loss of one AZ, the DNS record will automatically resolve to the IP of the healthy load balancer and the traffic will be dispatched to the healthy nodes of the other AZs, maintaining the availability of Kibana.
```